src/test/java/games/strategy/engine/data/annotations/ValidateAttachmentsTest.java

```diff
@@ -292,7 +292,7 @@ private static String validateAttachment(final Class<? extends IAttachment> claz
       // validate that there is a field and a getter
       Field field = null;
       try {
-        field = PropertyUtil.getFieldIncludingFromSuperClasses(clazz, "m_" + propertyName, false);
+        field = PropertyUtil.getFieldIncludingFromSuperClasses(clazz, propertyName, false);
```

Comment:
It looks like the `PropertyUtil` class handles searching for fields with or without the `m_` prefix via the `PropertyUtil#getPropertyFieldObject()`.  However, the `PropertyUtil#getFieldIncludingFromSuperClasses()` method, used in the above test code, only looks for a field with the given name without trying the additional `m_` prefix.  Unfortunately, it's not straightforward to swap one method call for the other.  In fact, it looks like it would require some significant refactoring to get this to work as expected.

With that said, I'm reconsidering my recommendation that the changes to the test attachment classes be reverted.  If `PropertyUtil` can handle either prefixed or unprefixed fields, then the test simply needs to be updated.  Granted, this change reports about 400 more warnings to the console than previously because a lot of the production attachment classes use the `m_` prefix for fields.  But it seems like the right thing to do is fix the test in a separate PR instead of reverting the changes in this package.

**TL;DR** The changes in the `src/test/java/games/strategy/engine/data/annotations` folder are fine.  The only thing that needs to be addressed is updating the comment in `InvalidFieldNameExample`.